spec.html

```diff
@@ -32544,67 +32544,60 @@ <h1>Array.prototype.some ( _callbackfn_ [ , _thisArg_ ] )</h1>
-        <p>If _comparefn_ is not *undefined* and is not a consistent comparison function for the elements of this array (see below), the sort order is implementation-defined. The sort order is also implementation-defined if _comparefn_ is *undefined* and SortCompare does not act as a consistent comparison function.</p>
-        <p>Let _proto_ be _obj_.[[GetPrototypeOf]](). If _proto_ is not *null* and there exists an integer _j_ such that all of the conditions below are satisfied then the sort order is implementation-defined:</p>
-        <ul>
-          <li>
-            _obj_ is sparse
-          </li>
-          <li>
-            0 &le; _j_ &lt; _len_
-          </li>
-          <li>
-            HasProperty(_proto_, ToString(_j_)) is *true*.
-          </li>
-        </ul>
-        <p>The sort order is also implementation-defined if _obj_ is sparse and any of the following conditions are true:</p>
+        <p>The sort order is implementation-defined if any of the following conditions are true:</p>
         <ul>
           <li>
-            IsExtensible(_obj_) is *false*.
+            If _comparefn_ is not *undefined* and is not a consistent comparison function for the elements of _items_ (see below).
           </li>
           <li>
-            Any integer index property of _obj_ whose name is a nonnegative integer less than _len_ is a data property whose [[Configurable]] attribute is *false*.
-          </li>
-        </ul>
-        <p>The sort order is also implementation-defined if any of the following conditions are true:</p>
-        <ul>
-          <li>
-            If _obj_ is an exotic object (including Proxy exotic objects) whose behaviour for [[Get]], [[Set]], [[Delete]], and [[GetOwnProperty]] is not the ordinary object implementation of these internal methods.
-          </li>
-          <li>
-            If any index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
-          </li>
-          <li>
-            If _comparefn_ is *undefined* and the application of ToString to any value passed as an argument to SortCompare modifies _obj_ or any object on _obj_'s prototype chain.
-          </li>
-          <li>
-            If _comparefn_ is *undefined* and all applications of ToString, to any specific value passed as an argument to SortCompare, do not produce the same result.
+            If _comparefn_ is *undefined*:
+            <ul>
+              <li>
+                SortCompare does not act as a consitent comparison function.
+              </li>
+              <li>
+                All applications of ToString, to any specific value passed as an argument to SortCompare, do not produce the same result.
+              </li>
```

Comment:
For clarity and simplicity, I'd avoid the nested list here and instead flatten it:

```
<li>If _comparefn_ is not *undefined* and is not a consistent comparison function for the elements of _items_ (see below).</li>
<li>If _comparefn_ is *undefined* and ...</li>
<li>If _comparefn_ is *undefined* and ...</li>
```

Reply:
Done